src/test/java/GameFieldTest.java

```diff
@@ -31,4 +36,21 @@ public void shouldMoveObjectWhenIsUpdated() throws Exception {
 
         assertEquals(expected, gameObject.getPos());
     }
+
+    @Test
+    public void shouldCallIntersectGameObjectWhenCollisionHappened() throws Exception {
+        GameField gameField = new GameField(new Point(0, 0), new Point(10, 10));
```

Comment:
please extract common part of test to
```
private GameField gameField;
@Before
public void setUp() {
}
```
method to avoid duplication and to facilitate test distinguishing